python/src/QuantileConfidence_doc.i.in

```diff
@@ -208,6 +208,48 @@ Examples
 
 // ---------------------------------------------------------------------
 
+%feature("docstring") OT::QuantileConfidence::computeUnilateralConfidenceIntervalWithCoverage
+"Evaluate an unilateral confidence interval of a quantile.
+
+Refer to :func:`computeUnilateralConfidenceInterval()`
+
+Parameters
+----------
+sample : 2-d sequence of float
+    Quantile level
+lower_bounded : bool, optional
+    True indicates the interval is bounded by a lower value.
+
+    False indicates the interval is bounded by an upper value.
+
+    Default value is False.
+
+Returns
+-------
+ci : :class:`~openturns.Interval`
+    Quantile confidence interval.
+
+coverage : float
+    The actual coverage probability of the confidence interval, i.e. the largest possible value of :math:`\beta`.
```

Comment:
This definition is confusing, because there is a difference between beta and the coverage. 
I suggest this:
```suggestion
If `lower_bounded` is True, then the `coverage` is the value of :math:`\proba{X_{(k_1)} \leq x_{\alpha} })`.
Otherwise, the `coverage` is the value of :math:`\proba{x_{\alpha} \leq X_{(k_2)}}`.
In both cases, the coverage is guaranteed to be greater or equal to :math:`\beta`.
```